examples/js/lines/LineSegments2.js

```diff
@@ -70,14 +70,21 @@ THREE.LineSegments2.prototype = Object.assign( Object.create( THREE.Mesh.prototy
 
 			}
 
+			var threshold = 0;
+			if ( 'Line2' in raycaster.params ) {
+
+				threshold = raycaster.params.Line2.threshold || 0.0;
+
+			}
```

Comment:
Does this pattern work?
```js
var threshold = ( raycaster.params.Line2 !== undefined ) ? raycaster.params.Line2.threshold || 0 : 0;
```

Reply:
It does -- I'd initially considered that but thought it was more difficult to read. If you'd prefer I can change it.

Reply:
It is not up to me, but that is the pattern we usually follow. /ping @Mugen87

Reply:
Got it -- done!